modules/users/tests/server/user-profile.server.routes.tests.js

```diff
@@ -572,43 +574,46 @@ describe('User profile CRUD tests', function () {
   });
 
   describe('Username Update', function () {
-    it('should update it', function (done) {
-      var user2 = _user;
-      user2.username = _user.username + '01';
-      delete user2.email;
+    it('should not let a new user to update', function (done) {
       agent.post('/api/auth/signin')
         .send(credentials)
         .expect(200)
         .end(function (err) {
           if (err) {
             return done(err);
           }
+          var user2 = _user;
+          user2.username = _user.username + '01';
+          delete user2.email;
           agent.put('/api/users')
             .send(user2)
-            .expect(200)
+            .expect(403)
             .end(function (err, res) {
               if (err) {
                 return done(err);
               }
-              res.body.username.should.equal(user2.username);
+              res.body.message.should.equal('You cannot change your username at this time.');
               return done();
             });
         });
     });
 
-    it('should fail to update if the three months have not passed by',
-      function (done) {
-        var user2 = _user;
-        delete user2.email;
+    it('should update it an user created 3 months ago', function (done) {
+      var threeMonthsAgo = moment(user.created)
+        .subtract(3, 'months')
+        .calendar();
+      user.update({ $set: { created: threeMonthsAgo } }, function (err) {
+        should.not.exist(err);
         agent.post('/api/auth/signin')
           .send(credentials)
           .expect(200)
           .end(function (err) {
             if (err) {
               return done(err);
             }
-            // First change
+            var user2 = _user;
             user2.username = _user.username + '01';
+            delete user2.email;
```

Comment:
Would it work to send only the username? 
```js
send({
  username: 'new01'
})
```

Reply:
It will work.